### PR TITLE
Support database names with tblank spaces

### PR DESCRIPTION
**Pull request by gacarrillor**
Hi @m-kuhn, our new colleague @jhonalex06 attempted to use Project Generator using a database name with blank spaces and it threw a Python error. So he fixed that in this PR. 

He also noticed that using databases without PostGIS in the Generate dialog (Source: 'PostGIS') was throwing an error, and then he also fixed it in this PR.

In summary, this PR:

 + Adds support for database names with blank spaces.
 + Handles the PostGIS doesn't exist case in Generate->Source:PostGIS.

### CODE REVIEW

File: projectgenerator/gui/generate_project.py [hunk 1]
```diff
@@ -262,6 +263,14 @@ def accepted(self):
                     self.progress_bar.hide()
                     return
 
+            if self.type_combo_box.currentData() == 'pg':
+                if not generator._postgis_exists():
+                    self.txtStdout.setText(
+                        self.tr('The current database does not have PostGIS installed! Please install it before proceeding.'))
```
[the remark this answers is not in the document]
Comment (gacarrillor) [reply projectgenerator/gui/generate_project.py]: Better!

[the remark this answers is not in the document]
Comment (m-kuhn) [reply projectgenerator/gui/generate_project.py]: Nice, this new beta feature :)

[the remark this answers is not in the document]
Comment (gacarrillor) [reply projectgenerator/gui/generate_project.py]: Oh, it was only to test the beta feature? :P

[the remark this answers is not in the document]
Comment (m-kuhn) [reply projectgenerator/gui/generate_project.py]: Yes, only for this! ;)

**Pull request merged**